src/App.css

```
@@ -0,0 +1,8 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: calc(100vh - 16px);
```

This leaves 16px on the bottom white for me, 

If there is no good reason to do this, then, 

```suggestion
  height: 100vh;
```